app/components/SnackbarToast.tsx

```diff
@@ -31,16 +34,42 @@ export default function SimpleSnackbar({
   );
 
   return (
-    <div>
-      <Snackbar
-        open={open}
-        autoHideDuration={3000}
-        anchorOrigin={{ vertical: "top", horizontal: "center" }}
-        onClose={handleClose}
+    <Snackbar
+      open={open}
+      autoHideDuration={3000}
+      anchorOrigin={{ vertical: "top", horizontal: "center" }} // Position at the top initially
+      onClose={handleClose}
+      TransitionComponent={Fade}
+      ContentProps={{
+        style: {
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+        },
+      }}
+      style={{
+        position: "fixed",
+        top: "50%",
+        left: "50%",
+        width: "100%",
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
+      <SnackbarContent
+        style={{
+          backgroundColor: "red",
```

Comment:
I think the red is a bit intense. What do you think about the yellow that we use for the progress bar? --colorFilledProgressBarColor: #fde55b;

Reply:
i think an error code needs to be red, but we can make it a lighter red